dali/operators/reader/nvdecoder/cuvideodecoder.cc

```diff
@@ -69,7 +69,7 @@ const char* GetVideoChromaFormatString(cudaVideoChromaFormat eChromaFormat) {
         { cudaVideoChromaFormat_444,        "YUV 444"              },
     };
 
-    if (static_cast<size_t>(eChromaFormat)
```

Comment:
Please revert - the underlying type of an `enum` is implementation-defined and might be signed.